events/views.py

```diff
@@ -219,3 +225,20 @@ def stats(request):
 
     return render_to_response(
         'stats/index.html', data, context_instance=RequestContext(request))
+
+def events_category(request, id, slug):
+    """
+        Filter evets for category
+    """
+    events = CategoryEvent.objects.filter(category_id=id)
```

Comment:
Here used try and catch why i need use filter model.

Used get_object_or_404 for model category.